tests/fetch/fetch-catch.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from "@playwright/test";
+
+test("Should be reject if native fetch is reject", async ({ page }) => {
+    await page.goto("http://127.0.0.1:8080/example/common.html");
+    const obj = {
+        isError: false,
+        isFinished: false
+    }
+    await page.evaluate((obj) => {
```

Comment:
Optional, since I’m not sure if it’s the same as puppeteer

but for cases like this, I’d usually return a value from evaluate 

in this case, just await (don’t need isFinished) return true or false (instead of isError)

Reply:
I tried to return true or false, but the return of the await is `{}`.

Reply:
changed in #150